client/styles/components/_sidebar.scss

```diff
@@ -286,8 +285,42 @@
   background-color: transparent;
   border: none;
 }
+.sidebar_file{
```

Comment:
since this is using the same code as the Nav, maybe this should be moved into a SASS placeholder or mixin?